documentation/user_docs/docs/index.rst

```diff
@@ -0,0 +1,162 @@
+.. Horace documentation master file, created by
```

Comment:
The "created by" and "you can adapt" comments should probably be removed.

This is the master file. It has been been adapted.